src/content/1.7/modules/concepts/_index.md

```diff
@@ -5,3 +5,23 @@ chapter: true
 ---
 
 # Extension concepts
+
+## Symfony extension concepts
+
+PrestaShop modules acts as Symfony bundles, extension points are added continuously in each minor version:
+
+| Version  | Symfony features                                               |
+|----------|----------------------------------------------------------------|
+| 1.7.3    | [Twig templates][1] and [Services][2]                          |
+| 1.7.4    | [Configuration Forms][3] and [Console commands][4]             |
+| 1.7.5    | [Modern controllers and Security][5]                           |
+| 1.7.6    | [Doctrine ORM][6] and [Entity forms][7]                        |
+| 1.7.7    | Help us improve the developer experience!                      |
```

Comment:
I suggest we either keep a list of "things/tables/lists we need to update at each version" or remove this part, because else I'm worried it will stay like this for 1.7.7, 1.7.8, 1.7.9 (if there is one) and so on 😅